Cargo.toml

```diff
@@ -1,9 +1,29 @@
 [workspace]
 
 members = [
-  "chaincash_server",
-  "chaincash_cli",
-  "chaincash_store",
-  "chaincash_app",
-  "chaincash_predicate"
+  "crates/chaincash_server",
+  "crates/chaincash_cli",
+  "crates/chaincash_store",
+  "crates/chaincash_app",
+  "crates/chaincash_predicate",
 ]
+
+[workspace.dependencies]
```

Comment:
Could we please only include dependencies that are used by more than one `chaincash` crate as workspace dependencies.

For example, the `diesel` crates will only be used in `chaincash_store` so lets move them back there